packages/dashboard/src/app/views/templateDetails/content/detailsGallery/index.js

```diff
@@ -106,9 +106,7 @@ function DetailsGallery({
         size={BUTTON_SIZES.SMALL}
         variant={BUTTON_VARIANTS.CIRCLE}
         aria-label={__('View previous template', 'web-stories')}
-        onClick={({ currentTarget }) => {
-          // blurring target here because memoized button remains active on click
-          currentTarget.blur();
```

Comment:
Buttons would still lose focus when clicked. Decided to remove this, even though the button will remain active.